src/main/java/duke/util/Save.java

```diff
@@ -0,0 +1,244 @@
+package duke.util;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.enums.CommandEnums;
+import duke.enums.Type;
+import duke.exceptions.InvalidCommandException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.Todo;
+
+/**
+ * Represents a handler to load a save file, or save to a file.
+ */
+public class Save {
```

Comment:
Perhaps can consider an alternative name for the class since the class also contains methods other than saving. Other than that, the code looks pretty good.